src/index.js

```diff
@@ -155,13 +155,16 @@ export default class Storage {
 
   /**
    * Request a model based on its identity (name)
-   * @param {string} id The identity of a registered model
-   * @return {Promise}  The model class
+   * @param {string|array} id The identity of a registered model, or array of identities
+   * @return {Promise}  The model class(es)
    * @example app.get('storage').getModel('user')
    */
   
   getModel (id) {
```

Comment:
I proposed `setModel` and `model` (don't prefix the common use), but really either way.